securedrop_client/gui/widgets.py

```diff
@@ -1873,7 +1873,8 @@ def _on_export_clicked(self):
             self.controller.sync_api()
             return
 
-        dialog = ExportDialog(self.controller, self.file.uuid)
+        dialog = ExportDialog(self.controller, self.file.uuid,
+                              self.file.original_filename)
```

Comment:
We could also change the signature and pass in the file object. I assumed there might be reasons not do so this (e.g. limit code exposure to all properties of the object), but happy to refactor. Note we'd have to mock or create file objects throughout the tests.

Reply:
(Chatted about this w/ Jen and she agreed that the function signature can stay as it is in this PR for now, but if we start throwing in more than 2 attributes, it may make sense to refactor to use the object reference instead.)